index.js

```diff
@@ -0,0 +1,138 @@
+module.exports = {
+  env: {
+    node: true,
+    es2020: true,
+    "jest/globals": true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+    "prettier/@typescript-eslint",
+    "plugin:jest/recommended",
+    "plugin:json/recommended",
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: "tsconfig.json",
```

Comment:
sometimes we have multiple tsconfigs. sometimes even one for parsing. can these finally go away with this?

Reply:
Yes, the separate config for linting is gone in all refactored projects. We still might have a separate one for pike in package repos, but that's fair game.